enginetest/queries/convert_queries.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2023 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package queries
+
+var ConvertTests = []struct {
+	Field   string
+	Op      string
+	Operand string
+	ExpCnt  int
+}{
+	{Field: "i8", Op: "=", Operand: "3720481604718463778705849469618542795", ExpCnt: 0},
```

Comment:
Also datetime types, to the extent those aren't covered elsewhere

Reply:
string and decimal lookups w/ convert definitely don't work, out of scope for fixing nautobot asap. ill add float tests because I supposedly fixed those

Reply:
i did add string/datetime -> numeric also. all types -> string/datetime left for a future PR